src/agent/base_agent.py

```diff
@@ -83,6 +83,8 @@ def __init__(
             max_new_tokens=300,
         )
         self.llm = OpenAI(model="gpt-3.5-turbo-instruct")
```

Comment:
Lets have llama3 8B as default model

Reply:
And then for line 80: multimodal model default should be set to Llava or even llava-gradient using ollama with option to change to gpt-4-preview